### ISSUE TITLE
creating multiple folder with UUID when mounting a drive

### ISSUE THREAD

**Issue by prabhu7x** [SL-RU/ranger_udisk_menu#10]
I am glad found this. but every time when mounting a drive it creating separate folder like UUIDx , [where x = 1,2,3,...] . Anyone know how to get rid of this. And this happens  after fresh reboot otherwise it mounting same folder.
![image](https://user-images.githubusercontent.com/112368340/229979056-b88d8416-b539-472a-bb5d-8e591a09c106.png)

**Comment by SL-RU** [comment]
Hello, mounting is accomplished using udiskctl2: https://github.com/SL-RU/ranger_udisk_menu/blob/master/menu.py#L208

 Sadly it has no options to set mount path. Why multiple directories are bothering you?

**Comment by prabhu7x** [comment]
not really. I am kind of person who keep files, scripts etc, clean and delete unnecessary files. After I found  your ' ranger_udisks_menu ' I am using it frequently because when comparing it to my little script, it has more functionality. But it keep increasing those folders which feels me  you know, and I though it was bug a or something, with the same thought, I showed up above issue.
                                                   **I really appreciate your work of this and thank you**

**Comment by prabhu7x** [comment]
> Hello, mounting is accomplished using udiskctl2: https://github.com/SL-RU/ranger_udisk_menu/blob/master/menu.py#L208
> 
> Sadly it has no options to set mount path. Why multiple directories are bothering you?

Not like that. I am kind of person who keep files, scripts etc, clean and delete unnecessary files. After I found your ' ranger_udisks_menu ' I am using it frequently because when comparing it to my little script, it has more functionality. But it keep increasing those folders which feels me you know, and I though it was bug a or something, with the same thought, I showed up above issue.
                                  **and once  again thank you this plugin**
